components/st-microelectronics/stm-api.stanza

```diff
@@ -142,7 +168,12 @@ public defn setup-clocks (stm:JITXObject, HSE-freq:Double, HSE-ppm:Double, HSE-s
     if supports?(instantiable-type(stm), high-freq-oscillator) and HSE-ppm < 0.03 :
       switch(HSE-source):
         "crystal" :
-          inst hf-xtal : ocdb/epson/TSX-3225/component(HSE-freq)
+          var xtal-inst = ocdb/abracon/AMB7/component
```

Comment:
this can be immutable

```
; supported crystal oscillator frequencies
val HSE-FREQS = [
  16.0e6,
  24.0e6,
  25.0e6,
  26.0e6,
  32.0e6,
]
inst hf-xtal:
  if contains?(HSE-FREQS, HSE-freq): 
    ocdb/epson/TSX-322f/component(HSE-freq)
  else:
    ocdb/abracon/AMB7/component
if contains?(HSE-FREQS, HSE-freq):
  net(hf-xtal.gnd, gnd-pin)
  
```